x-pack/plugins/security/public/views/management/edit_role/components/privileges/es/elasticsearch_privileges.tsx

```diff
@@ -231,4 +232,17 @@ export class ElasticsearchPrivileges extends Component<Props, {}> {
 
     this.props.onChange(role);
   };
+
+  public onCreateRunAsOption = (option: any) => {
```

Comment:
__issue:__ hmm,  here is what I see if I click on "suggestion":

![bug](https://user-images.githubusercontent.com/1713708/54204084-d0993300-44d3-11e9-8b75-e7006667bc7f.gif)

I'd expect custom value to either be dismissed entirely or be moved after chosen suggestion and stay in "edit" mode.

I see the same behavior at https://elastic.github.io/eui/#/forms/combo-box, @chandlerprall is it expected to work like that with `onCreateOption`?

Reply:
This is a fun one, thanks for catching this @azasypkin. I'm only seeing this when starting to type what would appear as the very first item in the drop down and selecting it, if I try this with "ki" and select `kibana`, I don't see this occur, instead it inputs "ki" into the combo box, which is the same behavior that the index pattern selection performs:

![run-as-arbitrary](https://user-images.githubusercontent.com/627123/54207524-8c883d00-4497-11e9-8c7f-0b45e1229317.gif)